backend/app/entity/shortlink.go

```diff
@@ -17,3 +17,28 @@ type ShortLink struct {
 	OpenGraphTags metatag.OpenGraph
 	TwitterTags   metatag.Twitter
 }
+
+// ShortLinkInput represents possible ShortLink attributes for a new short link or an updated short link.
```

Comment:
```suggestion
// ShortLinkInput represents possible ShortLink attributes a short link.
```
We don't mention how we are going to use it here. That's the job and freedom of use case.